charts/neuvector/100.0.0+up2.2.0/Chart.yaml

```diff
@@ -2,12 +2,12 @@ annotations:
   catalog.cattle.io/auto-install: neuvector-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: NeuVector
-  catalog.cattle.io/kube-version: '>=1.18.0-0'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.22.0-0'
```

Comment:
Neuvector works with 1.22+, I think all the way up to 1.24 and beyond.

Reply:
This version specifically @selvamt94 confirmed it supports 1.23, so I'm waiting on other reviewers to update it the PR for everything together.